pkg/resources/manifests.go

```diff
@@ -270,6 +270,32 @@ func GenerateDeploymentManifest(ctx context.Context, workspaceObj *kaitov1alpha1
 	labelselector := &v1.LabelSelector{
 		MatchLabels: selector,
 	}
+	initContaiers := []corev1.Container{}
+	envs := []corev1.EnvVar{}
+
+	if len(workspaceObj.Inference.Adapters) != 0 {
+		for i, adapter := range workspaceObj.Inference.Adapters {
+			initContaier := corev1.Container{
+				Name:            fmt.Sprintf("init-container-%d", i+1),
+				Image:           adapter.Source.Image,
+				Command:         []string{"/bin/sh", "-c", fmt.Sprintf("mkdir -p /dev/shm/%s && cp -r * /dev/shm/%s", adapter.Source.Name, adapter.Source.Name)},
+				VolumeMounts:    volumeMount,
```

Comment:
Let us not using /dev/shm as the mount path for data.  It is the mount path for real shared memory between processes.